servicebox-app/src/test/java/io/halkyon/ClustersEndpointTest.java

```diff
@@ -0,0 +1,71 @@
+package io.halkyon;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.core.IsNot.not;
+
+import javax.ws.rs.core.MediaType;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class ClustersEndpointTest {
```

Comment:
Very minor remark: I think the tests for others entities are named in singular. To be homogeneous it should be good also here to call it ClusterEndpointTest or rename others too

Reply:
I think it makes more sense to rename others to the plural, as we're covering several endpoints.